en/docs/get-started/identity-anti-patterns-and-the-identity-bus.md

```diff
@@ -90,13 +90,13 @@ pattern.
 5.  **Role mapping.** Your service provider needs to authorize users
     once they are logged in. What the user can do at the identity
     provider is different from what the same user can do at the service
-    provider. User's roles from the identity provider define what he can
+    provider. Users' roles from the identity provider define what they can
```

Comment:
The user's role ?

Reply:
then we would have to use he or she instead of they which is not recommended. Is that okay?